@here/olp-sdk-dataservice-read/lib/client/DataRequest.ts

```diff
@@ -101,6 +104,9 @@ export class DataRequest {
     }
 
     /**
+     * @deprecated Please ignore this param and use latest version of the layer.
+     * In case of using deprecated constructor should be added to this method.
```

Comment:
should not be added. we just ignoring that in the versioned layer client. The latest version will be used in any cases if a user will use deprecated constructor